Added LLS request form (#6250)
perma-id/w3id.org: molepi/Forms/Data_requests/LLS/readme.md

# Leiden Longevity Study Data request form

> For the Leiden Longevity Study, long-lived siblings of European descent were recruited together with their offspring and the partners of this offspring between 2002 and 2006. Families were recruited if at least two long-lived siblings were alive and fulfilled the age criterion of 89 years or older for males and 91 years or older for females, representing less than 0.5% of the Dutch population in 2001 (Schoenmaker et al 2006, doi: 10.1038/sj.ejhg.5201508; Westendorp et al 2009, doi: 10.1111/j.1532-5415.2009.02381.x). In total 944 long-lived siblings were included with a mean age of 94 years (range 89-104), 1671 offspring (61 years, 34-81) and 744 partners (60 years, 30-79). It is family based study consists of 1671 offspring of 421 nonagenarians sibling pairs of Dutch descent, and their 744 partners. Of all LLS participants HRC1.1 imputed genotypes are available.

Reviewer: ```suggestion
For the Leiden Longevity Study, long-lived siblings of European descent were recruited, together with their offspring and the partners of this offspring, between 2002 and 2006. Families were recruited if at least two long-lived siblings were alive and fulfilled the age criterion of 89 years or older for males and 91 years or older for females, representing less than 0.5% of the Dutch population in 2001 (Schoenmaker et al 2006, doi: 10.1038/sj.ejhg.5201508; Westendorp et al 2009, doi: 10.1111/j.1532-5415.2009.02381.x). In total, 944 long-lived siblings were included with a mean age of 94 years (range 89-104), 1671 offspring (61 years, 34-81) and 744 partners (60 years, 30-79). Its family based study consists of 1671 offspring of 421 nonagenarians sibling pairs of Dutch descent, and their 744 partners. HRC1.1 imputed genotypes of all LLS participants are available.
```

## Contact

This w3id is administered by:
Julian Dekker
J.J.Dekker1@lumc.nl
GitHub username: JulianDekker

> 

Reviewer: ```suggestion
This w3id is administered by:

- Julian Dekker
  - J.J.Dekker1@lumc.nl
  - GitHub username: JulianDekker
```
